src/components/CreatePost/Transportation.tsx

```diff
@@ -0,0 +1,50 @@
+import DirectionsBusIcon from '@mui/icons-material/DirectionsBus';
+import DirectionsCarIcon from '@mui/icons-material/DirectionsCar';
+import DirectionsRunIcon from '@mui/icons-material/DirectionsRun';
+import FlightIcon from '@mui/icons-material/Flight';
+import PedalBikeIcon from '@mui/icons-material/PedalBike';
+import SailingIcon from '@mui/icons-material/Sailing';
+import TrainIcon from '@mui/icons-material/Train';
```

Comment:
```suggestion
import {
  DirectionsBus,
  DirectionsCar,
  DirectionsRun,
  Flight,
  PedalBike,
  Sailing,
  Train,
} from '@mui/icons-material';
```
`Icon`을 붙이지 않는다면 이런식으로 한 번에 가져올 수 있을 것 같은데`Icon`을 붙이는 이유가 있으신가요?

Reply:
오 뒤에 Icon을 빼야 되는거였군요...
```js
import {
  DirectionsBusIcon
} from '@mui/icons-material';
```

이런식으로 해서 안됐었는데 감사합니다!